### ISSUE TITLE
Updating your links to the seqan packages on Fedora

### ISSUE THREAD

**Issue by sagitter** [seqan/seqan#2222]
Hi all.

[Your online documentation](http://seqan.readthedocs.io/en/master/Infrastructure/Use/Install.html#native-package-management) points to the [seqan web-page](https://src.fedoraproject.org/rpms/seqan) available in the Fedora Project. Until now, only the `seqan-1.4.*` rpms were distributed because needed by [OpenMS](https://src.fedoraproject.org/rpms/openms).

I have rebuilt newer versions of Seqan (2.* release) as [`seqan2` rpms](https://src.fedoraproject.org/rpms/seqan2) in order to provide them to the Fedora end-users and developers.

To summarize, on Fedora >= 26 `seqan` and `seqan2*` coexist:

`seqan-devel` and `seqan2-devel` libraries can be installed together.
`seqan2` (rpm containing applications only) make obsolete `seqan`.

Please, update your list of native packages.

**Comment by h-2** [comment]
Thank you for your porting efforts!

> seqan-devel and seqan2-devel libraries can be installed together.
seqan2 (rpm containing applications only) make obsolete seqan.

This is of course up to you as the maintainer, but we really prefer the applications not be called just `seqan`, because it confuses people. "SeqAn" is the (header-only) library. The "SeqAn project" also encompasses applications that are usually called "SeqAn Apps".
In any case the library is not the developer package of the apps, third party apps use it, as well! Beginning with SeqAn3 all apps are dropped from the repository, there is only the library and all apps are treated as third party and shipped individually. But if it was like this before in fedora, there might be some point to consistency...

**Comment by sagitter** [comment]
Is this rpms tree better for you?

```
 seqan
    |
    |-- seqan2-apps
    |-- seqan2-headers
    |-- seqan2-doc
    |-- seqan2-examples
```

**Comment by h-2** [comment]
I would prefer
```
 seqan
    |
    |-- seqan-apps      [replaces the old package so doesn't need to be versioned]
    |-- seqan2          [library]
    |-- seqan2-doc
    |-- seqan2-examples
```

This also makes it clearer that -doc and -examples refer to the library, and are not related to the apps.

If that's not possible:

```
 seqan
    |
    |-- seqan2-apps  
    |-- seqan2-library
    |-- seqan2-doc
    |-- seqan2-examples
```

But, like I said, I don't know any conventions you might have at Fedora (e.g. Debian requires libraries to have a `lib`-prefix).

**Comment by h-2** [comment]
@sagitter If you have decided on names for the packages, I can immediately update the documentation!

**Comment by h-2** [comment]
@sagitter Please decide on this ASAP so I can update the documentation shipped with the next release, to tagged on monday! Thanks!

**Comment by sagitter** [comment]
`seqan2` is currently provided as

```
seqan2 (source)
seqan2-apps
seqan2-headers
seqan2-doc
seqan2-examples
```

https://koji.fedoraproject.org/koji/buildinfo?buildID=1017831
